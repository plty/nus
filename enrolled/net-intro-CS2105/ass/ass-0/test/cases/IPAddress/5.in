00111111000111110111111111111111
