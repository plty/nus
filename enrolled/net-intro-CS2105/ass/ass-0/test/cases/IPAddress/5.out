63.31.127.255
